core/modules/node/node.module

```diff
@@ -2014,9 +2010,35 @@ function node_block_configure($delta = '', $settings = array()) {
       '#options' => backdrop_map_assoc(array(2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 25, 30)),
     );
   }
+  if ($delta == 'syndicate') {
+    $form['syndicate_settings'] = array(
+      '#type' => 'radios',
+      '#title' => t('Choose the linked feed'),
+      '#default_value' => isset($settings['syndicate_settings']) ? $settings['syndicate_settings'] : 'promoted',
+      '#options' => array(
+        'promoted' => t('Always show RSS link for promoted content'),
+        'page_content' => t('Show RSS link for the current page, if available'),
+      ),
+    );
+  }
   return $form;
 }
 
+/**
+ * Returns a themed RSS icon which links to a provided feed.
+ */
+function node_syndicate_content($settings) {
+  if (isset($settings['syndicate_settings']) && $settings['syndicate_settings'] == 'page_content') {
+    return backdrop_get_feeds();
+  }
+  // Default to promoted content to support legacy Syndicate blocks.
+  return array(
+    '#theme' => 'feed_icon',
+    '#url' => 'rss.xml',
+    '#title' => t('Syndicate'),
```

Comment:
the `#title` is used to create feed icon title attribute, the this one generate a tooltip with "Subscribe to Syndicate", maybe this PR is a good occasion to do better, something like "Subscribe to promoted content (feed)".

Reply:
Even just "Subscribe" would be better than "Syndicate". Would anyone miss the previous language here?